### DISCUSSION TITLE
Copilot 1.83.41 and Remote respository

### DISCUSSION THREAD

**Discussion by perrintong325** [community/community#53337] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi

My copilot was just updated to 1.83.41 but it stopped working in the remote repository(SSH FS) but still works locally, rolled back to the previous version and all is good now. Any idea what could be causing this in the latest release? Thanks

**Comment by SebMoore** [comment]
Same issue here! Newest version (1.83.41) does not launch if a remote folder is added to the workspace using the VSCode extension SSH FS.
- The Copilot icon does not appear in the status bar
- Autocompletion does not occur
- Trying to open the Completions panel throws an error that the command was not found
- Nothing appears in the output tab for Github Copilot

Rolling back to 1.82.15 (previous version) fixes the issue.

**Reply by desh2608** [reply]
Had the same issue, and fixed by rolling back to the previous version.

**Comment by Akash1134** [comment]
Hi All,

I can confirm that this issue of Copilot v1.83 not working with certain VSCode extension SSH FS has been identified by our team. (I just got back from a week-long out-of-office, the issue was actually identified a few days ago)

Rolling back Copilot to version Rolling back to 1.82.x is currently a workaround but we are working towards a fix. 

Thanks!

**Reply by perrintong325** [reply]
Hi, is there any updates on this especially since the new Copilot nightly is having the same issues and I haven't been able to test out the new chat beta

**Comment by LukePrior** [comment]
Has anyone found a fix for this?

**Reply by perrintong325** [reply]
No, I'm using macFuse to mount it to my computer instead before there's a fix

**Comment by perrintong325** [comment]
@Akash1134 hi is there any updates?

**Comment by tpope** [comment, marked as answer]
This is fixed in 1.109.
